pages/index.tsx

```diff
@@ -28,7 +30,9 @@ export default function Home() {
             <Btn label="Go!" />
           </div>
         </div>
-        <footer className={styles.footer}></footer>
+        <footer className={styles.footer}>
+          <Footer homeIcon="/home.svg" favsIcon="/heartf.svg" />
+        </footer>
```

Comment:
To me it looks like your ```<Footer /> ``` component is bringing its own footer wrapper tag ... you have a double footer encapsulation in my opinion. can you check it out?